Makefile

```diff
@@ -117,8 +117,18 @@ format: pre-format
 	@cargo fmt --all -- --check >/dev/null || \
 	cargo fmt --all
 
+LATEST_AUDIT_VERSION = $(strip $(shell cargo search cargo-audit | head -n 1 | awk '{ gsub(/"/, "", $$3); print $$3 }'))
```

Comment:
seems other variables all at the beginning of this file, maybe we should them consistent